### ISSUE TITLE
two app with same account

### ISSUE THREAD

**Issue by moshtagh** [Udinic/AccountAuthenticator#11]
If we install two sample application on single adndroid device, only the first one can create account.
It seems that application with shared account should use same "android:sharedUserId".

**Comment by ahadxfighter** [comment]
you mean only one account can be signed in  or be used at a time?

**Comment by moshtagh** [comment]
no, I mean only the first installed app has permission to show login activity and create account; try install both sample apps in Udinic/AccoutAuthentictor, then use both of them separately to create account. Only one app can do.

**Comment by ahadxfighter** [comment]
yes. I followed his code, and it is working fine. But since firebase out there, it does not seem helpful managing account for your app. What do you think?

**Comment by moshtagh** [comment]
Please take a look at [https://www.captechconsulting.com/blogs/Android-Single-Account-Multiple-Application-Prescription]
I think is dose not matter whether use of firebase or not.

**Comment by santoki-darshan** [comment]
I don't know what happened but it wasn't working yesterday but it's working fine today with two apps having the same account.

**Comment by 1136066873** [comment]
测试测试。

**Comment by 1136066873** [comment]
I just want to know how you use this AccountManager on multiple applications, because I found that if both applications depend on the AccountAuthenticator library and the two application signatures are different, then in the first application After logging in to the successful account and saving the token, in the second application, it is impossible to obtain any token after the first application login successfully in AccountAuthenticator, but when I sign the same signature information for the two applications, this is OK. of. Both applications can share account information together. Have you ever encountered this problem?

**Comment by santoki-darshan** [comment]
Yeah, that's true.
